### PR TITLE
Doc'd send_email() behavior when from_email is None.

### PR DESCRIPTION
**Pull request by AdrianDAlessandro**
Just a hole I found in the documentation that would've helped me get to my solution faster.

### CODE REVIEW

File: docs/topics/email.txt [hunk 1]
```diff
@@ -51,7 +51,7 @@ are required.
 
 * ``subject``: A string.
 * ``message``: A string.
-* ``from_email``: A string.
+* ``from_email``: A string. If ``None`` uses the DEFAULT_FROM_EMAIL.
```
Comment (pope1ni) [inline docs/topics/email.txt:54]: Maybe change to the following to match similar messages below:
```
If ``None``, Django will use the value of the :setting:`DEFAULT_FROM_EMAIL` setting.
```

Comment (AdrianDAlessandro) [reply docs/topics/email.txt]: Thanks, done.

File: docs/topics/email.txt [hunk 2]
```diff
@@ -51,7 +51,7 @@ are required.
 
 * ``subject``: A string.
 * ``message``: A string.
-* ``from_email``: A string.
+* ``from_email``: A string. If ``None``, Django will use the value of the :setting:`DEFAULT_FROM_EMAIL` setting.
```
Comment (pope1ni) [inline docs/topics/email.txt:54]: Please can you also wrap to 80 characters like the existing text below?

**Comment by felixxm** [comment]
@AdrianDAlessandro Thanks :+1: Welcome aboard :boat:

**Pull request merged**